app/graphql/resolvers/concerns/collection.rb

```diff
@@ -6,11 +6,14 @@ module Concerns
     module Collection
       extend ActiveSupport::Concern
 
+      include Sorting
+
       included do
         type ["Types::#{self::MODEL_CLASS}".safe_constantize], null: false
         argument :search, String, 'Search query', required: false
         argument :limit, Integer, 'Pagination limit', required: false
         argument :offset, Integer, 'Pagination offset', required: false
+        argument :sort_by, [String], 'Sort results', required: false
```

Comment:
You can define a [union type](https://graphql-ruby.org/type_definitions/unions#defining-union-types). Please add a `BaseUnion` type as mentioned in that doc and then a `StringOrArrayOfStrings` union type, used like:

```suggestion
        argument :sort_by, StringOrArrayOfStrings, 'Sort results', required: false
```

Reply:
Nope, union doesn't allow primitive types https://github.com/rmosolgo/graphql-ruby/issues/530

Reply:
@akofink how bad it is if we only allow arrays of strings? :see_no_evil:

Reply:
For GraphQL, I'm not too worried about it, since we're just targeting our own UI. For REST, IMO it's a UX concern, but it looks like there's no problem with that implementation :+1: